### DISCUSSION TITLE
Can you end a phase and proceed to the next player

### DISCUSSION THREAD

**Discussion by Jon-115** [boardgameio/boardgame.io#1113] [General]
I begin a phase once a card has been drawn from the deck. Then the player can use the card or pass it to the next player. For my example each player passes and the phase is over, but instead of it still being the player who drew the cards turn, I want to end their turn and let the next player draw a card. I've tried to end the phase and then call end turn with the key next and the correct value, but it remains on the player that initiated the phase. Just wondering if I am able to change the player once the phase has ended.

**Comment by delucis** [comment]
You can’t use `endTurn` after using `endPhase`, but you can use a custom turn order to control how the first player of a phase is selected: https://boardgame.io/documentation/#/turn-order?id=creating-a-custom-turn-order
